files/setup.sh

```diff
@@ -37,6 +37,7 @@ else
     VCENTER_USERNAME_PROPERTY=$(vmtoolsd --cmd "info-get guestinfo.ovfEnv" | grep "guestinfo.vcenter_username")
```

Comment:
All these files are from a different commit, please force push without this one.